src/server/editorServices.ts

```diff
@@ -294,6 +304,7 @@ namespace ts.server {
         hostInfo: string;
         extraFileExtensions?: FileExtensionInfo[];
         watchOptions?: WatchOptions;
+        typeAcquisition?: TypeAcquisition;

```

Comment:
You shouldn’t need this since the options are per project and there isn’t anything in host